shared/src/commonTest/kotlin/co/nimblehq/mark/kmmic/data/repository/UserRepositoryTest.kt

```diff
@@ -0,0 +1,69 @@
+package co.nimblehq.mark.kmmic.data.repository
+
+import co.nimblehq.mark.kmmic.data.service.user.UserService
+import co.nimblehq.mark.kmmic.data.service.user.model.UserApiModel
+import co.nimblehq.mark.kmmic.domain.model.User
+import co.nimblehq.mark.kmmic.domain.repository.UserRepository
+import kotlin.test.BeforeTest
+import io.kotest.matchers.shouldBe
+import io.mockative.Mock
+import io.mockative.classOf
+import io.mockative.given
+import io.mockative.mock
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flow
+import org.koin.core.context.startKoin
+import org.koin.dsl.module
+import kotlin.test.Test
+import kotlinx.coroutines.test.runTest
+import org.koin.core.context.stopKoin
+import kotlin.test.AfterTest
+
+@ExperimentalCoroutinesApi
+class UserRepositoryTest {
+    @Mock
```

Comment:
It'd be nice to have a line break after the class declaration following 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 convention. This should apply to all the other classes such as repositories, services, etc.

Reply:
@AVI5HEK updated at [e796eb2](https://github.com/markgravity/kmm-ic/pull/72/commits/e796eb2998575c9a9fa82a0f9041c6b3db961ce0)